src/main/msp/msp.c

```diff
@@ -3646,7 +3646,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         int32_t ned_vel_east = (int32_t)sbufReadU32(src);   // ned_vel_east
         gpsSol.groundSpeed = (uint16_t)sqrtf((ned_vel_north * ned_vel_north) + (ned_vel_east * ned_vel_east));
         (void)sbufReadU32(src);             // ned_vel_down
-        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360);   // ground_course
+        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360) * 10;   // ground_course
```

Comment:
We can also do this to get 10x more resolution, afaik.
```suggestion
        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 3600);   // ground_course
```

Reply:
In the discord example, the Remote ID unit was logged and definitely was sending whole degrees, not decidegrees.
Do we have a "standard" on this where all Remote ID software must report data in a certain way?

Reply:
- prefer a standard, but RID modules in the wild already.  vendors will need to accommodate appropriately.
- can it be coded for if/then to decide if data is degrees or decidegrees?

Reply:
Looks like iNav expects centidegrees as the input value, I guess we should be consistent with their implementation.